### ISSUE TITLE
Effect_value_interpretation

### ISSUE THREAD

**Issue by StefanoCast** [ggloor/ALDEx_bioc#39]
Hi,
I am just starting to using ALDEx2 on my metagenomic datasets: it is really easy to implement! I am wondering about the interpretation of "effect" values, as calculated by the "aldex" function in a two group comparison of a metagenomic feature table.  
For example, a feature with "rab.all = -2.7", "rab.win.0 = -1.21", "rab.win.1 = -9.3", "diff.btw= -7.9", "diff.win = 3.4" and "effect = -2.1", is inferred to be highly abundant in the class "0" (given that "effect" is negative), or not? Corrected p-values for the feature are both < 0.05.
Sample size of class "1" is 21, while size of class "0" is 13.

Then, does a positive effect mean a major abundance of such feature in the class "1"? Which is the optimal "effect" cutoff (in absolute value)? Should I at the same time consider the "overlap" and corrected p-values for the identification of differentially abundant taxa? 
Thanks a lot,
best regards.

Stefano

**Comment by ggloor** [comment]
Hi Stefano
I'm attaching a preprint that will be submitted soon that will answer this question.

TLDR: an absolute effect of >2 is highly different and will almost certainly reproduce if you replicate the experiment; i.e., it is a real difference, and a very large one!

[peerj-gloor.pdf](https://github.com/ggloor/ALDEx_bioc/files/6317922/peerj-gloor.pdf)

**Comment by StefanoCast** [comment]
Thank you very much for the reply, your manuscript is very helpful!

Stefano
